src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -129,7 +129,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 debugPlusMode:=False,
                 xmlReferenceResolver:=xmlReferenceResolver,
                 sourceReferenceResolver:=sourceReferenceResolver,
-                syntaxTreeOptionsProvider:=Nothing,
+                SyntaxTreeOptionsProvider:=Nothing,
```

Comment:
I didn't do this change 😨 
Could be pretty lister did that while I was in a non-compiling state.